### ISSUE TITLE
6.12.1 fails to find pagmo and flann

### ISSUE THREAD

**Issue by yurivict** [dartsim/dart#1655]
pagmo2-2.18.0 and flann-1.9.1.29 are installed but it fails to find them:
```
CMake Warning at cmake/DARTMacros.cmake:145 (message):
  Looking for pagmo - NOT found, to use dart-optimizer-pagmo, please install
  pagmo
Call Stack (most recent call first):
  dart/optimizer/pagmo/CMakeLists.txt:3 (dart_check_optional_package)


-- Performing Test BT_USE_DOUBLE_PRECISION
-- Performing Test BT_USE_DOUBLE_PRECISION - Failed
CMake Warning at cmake/DARTMacros.cmake:145 (message):
  Looking for flann - NOT found, to use dart-planning, please install flann
Call Stack (most recent call first):
  dart/planning/CMakeLists.txt:3 (dart_check_optional_package)

```

**Comment by olzhas** [comment]
what OS are you using and how did you install those packages?

**Comment by yurivict** [comment]
FreeBSD.

All packages are installed from packages.

**Comment by olzhas** [comment]
can't help with this, I don't have FreeBSD at my hand.

**Comment by jslee02** [comment]
No plan to fix this for DART 6.12.1, but finding pagmo2 is fixed in `master` (DART 7). FLANN is no longer a dependency for DART 7.
